kinds/structured/team/team_kind.cue

```diff
@@ -0,0 +1,34 @@
+package kind
+
+name: "Team"
+maturity: "merged"
+
+lineage: seqs: [
+	{
+		schemas: [
+			// v0.0
+			{
+				// OrgId is the ID of an organisation the team belongs to.
+				orgId?: int64 @grafanamaturity(ToMetadata="sys")
+				// Name of the team.
+				name: string
+				// Email of the team.
+				email?: string
+				// AvatarUrl is the team's avatar URL.
+				avatarUrl: string @grafanamaturity(MaybeRemove)
+				// MemberCount is the number of the team members.
+				memberCount: int64 @grafanamaturity(ToMetadata="kind")
+				// TODO - it seems it's a team_member.permission, unlikely it should belong to the team kind
+				permission: #Permission @grafanamaturity(ToMetadata="kind", MaybeRemove)
+				// AccessControl metadata associated with a given resource.
+				accessControl: [string]: bool @grafanamaturity(ToMetadata="sys")
+				// Created indicates when the team was created.
+				created: string @grafanamaturity(ToMetadata="sys")
```

Comment:
Should it be `string` or `int64`?

Reply:
oh right, forgot about this! tl;dr - for now, the right answer is `int64`.

I _think_ the ideal thing here is a `time.Time`, as CUE has a native type for that - 
But, if we chase down `time.Time` a bit - that causes both Go and TS type generation to fail. The Go one currently fails because it means there's a `time` import that has to be detected:

```
generate code from .cue files
go generate ./pkg/plugins/plugindef
plugindef jenny pipeline failed: 1 error occurred:
	* PluginGoTypes: goimports added the following import statements to plugindef_type_gen.go:
	"encoding/json"
	"fmt"
Relying on goimports to find imports significantly slows down code generation. Either add these imports with an AST manipulation in cfg.ApplyFuncs, or set cfg.IgnoreDiscoveredImports to true
```

That's pretty easily fixed in thema, and i'll push such a fix presently. But once it's in place, the TS generation then fails with some real nastiness:

```
core kinddirs codegen failed: 2 errors occurred:
	* LatestMajorsOrXJenny: TSTypesJenny jenny failed on 0.0 schema for Team: generating TS for schema root failed: could not generate field: %!w(*errors.posError=&{{0xc000805dc0 14180} [] {bounds constraints are not supported as they lack a direct typescript equivalent []}}) for input "team"
	* TSVeneerIndexJenny: Team: generating TS for schema root failed: could not generate field: %!w(*errors.posError=&{{0xc000805dc0 14180} [] {bounds constraints are not supported as they lack a direct typescript equivalent []}})
```

That's a fix needed in cuetsy. (Another code generator TODO issue to be posted!)